prody/utilities/catchall.py

```diff
@@ -63,14 +63,12 @@ def calcModeClusters(similarities, n_clusters=0, linkage='all', method='tsne', c
         
 
     # Check inputs to make sure are of valid types/values
-    from prody.utilities import isListLike
-
     if not isinstance(similarities, np.ndarray):
         try:
             import pandas as pd
             if not isinstance(similarities, pd.DataFrame):
                 raise TypeError('similarities should be an instance of ndarray or DataFrame')
-        except:
+        except ImportError:
```

Comment:
That doesn't make sense. If there's an ImportError then it's not a TypeError. The TypeError goes with the np.ndarray check and should be back-indented. Something else should go here such as `pass`.

Reply:
It still might be easier to just remove pandas and have the user convert to numpy like they do for everything else. You could also have a variable marking if pandas is imported and do the type checking with that rather than coupled to the importing maybe.

Reply:
> It still might be easier to just remove pandas and have the user convert to numpy like they do for everything else. You could also have a variable marking if pandas is imported and do the type checking with that rather than coupled to the importing maybe.

Yes, I second this.